src/controllers/user.ts

```diff
@@ -0,0 +1,28 @@
+import express, { Request, Response } from 'express'
+
+import BaseController from './base'
+import UserModel from '../models/user'
+
+class UserController implements BaseController {
+
+  public path = '/api/users'
```

Comment:
Would it make sense to instead use graphql or is it overkill if we only do simple search?

Reply:
@domoritz I haven't used it before but I think querying with mongoose is anyway quite easy and has a good performance so it will be fine. GraphQL just provides a more intuitive way of querying data.

Reply:
Yeah, GraphQL is only an API for the client. I don't know whether it's overkill for us but it might be interesting.